webhook.go

```diff
@@ -12,6 +12,12 @@ var (
 	ErrInvalidInput = fmt.Errorf("invalid input")
 )
 
+type Register interface {
+	GetId() string
+	GetPartnerIds() []string
```

Comment:
```suggestion
```

i think we should remove `GetPartnerIds`, mainly because it's not webhook-schema abstraction but a argus/ancla abstraction

imo, `GetId` and `GetUntil` are great because the actual values in v1 and v2 are very different so having those getters help abstract that away.